acac_frontend/.graphqlconfig

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Gymkhana GraphQL Schema",
+  "schemaPath": "./src/plugins/schema.json",
+  "extensions": {
+    "endpoints": {
+      "Gymkhana Dev GraphQL Endpoint": {
+        "url": "http://localhost:8000/graphql",
```

Comment:
For now we can move forward with this. Afterwards make the base url dynamic (using env variables)

Reply:
yeah